omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -125,6 +126,20 @@ public void shouldVoidAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldListAllPatientIdentifiersWithVoidedIdentifiersForAPatient() throws Exception {
+		assertEquals(false, service.getPatientIdentifierByUuid(getUuid()).isVoided());
+		MockHttpServletRequest req = request(RequestMethod.DELETE, getURI() + "/" + getUuid());
+		req.addParameter("!purge", "");
```

Comment:
What is the use of the above parameter on line 132?

Reply:
Void, I it guess can be removed.

Reply:
Do you mean it is the way of making the request void the data?